docs/quickstarts/quickstart_flink_processor.md

```diff
@@ -21,15 +21,15 @@ You can start a Flink standalone cluster in your local environment with the foll
 command.
 
 ```bash
-$ ./flink-1.15.1/bin/start-cluster.sh
+$ ./flink-1.15.2/bin/start-cluster.sh
 ```
 
 You should be able to navigate to the web UI at [localhost:8081](http://localhost:8081)
 to view the Flink dashboard and see that the cluster is up and running.
 
 ### Run demo
 Execute the following command to run the
-[nyc_taxi_flink.py](../python/feathub/examples/nyc_taxi_flink.py) demo.
+[nyc_taxi_flink.py](../../python/feathub/examples/nyc_taxi_flink.py) demo.
```

Comment:
How about we rename `nyc_taxi_flink.py` to `nyc_taxi_flink_session.py` for consistency with `streaming_average_flink_cli.py`?